### PR TITLE
Pr/merge selection

### PR DESCRIPTION
**Pull request by stewue**

### CODE REVIEW

File: packages/merge/src/components/MergeTable/MergeErrors.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+              <Error key={`paths-${name}-${validator}-${index}`}>{sourceData.labels[name]}: {msg}</Error>
+            )
+          ))
+
+        const entityValidators = Object.keys(e.entityValidatorErrors)
+          .map(validator => e.entityValidatorErrors[validator].map((msg, index) =>
+            <Error key={`entity-${validator}-${index}`}>{msg}</Error>
+          ))
+
+        return fieldValidators.concat(entityValidators)
```
Comment (MethodenMann) [inline packages/merge/src/components/MergeTable/MergeErrors.js:34]: i prefer

`return [...fieldValidators, ...entityValidators]`

but of course, concat can be used as well

File: packages/merge/src/components/MergeTable/MergeTable.js [hunk 2]
```diff
@@ -59,11 +60,19 @@ const MergeTable = ({sourceData, executeMerge}) => {
       ? getColumnDefinition(sourceData, ColumnHeaderRendererContainer, CellRendererContainer, LabelCellRenderer) : []
   , [sourceData])
 
+  if (mergePending) {
+    return <LoadMask/>
```
Comment (MethodenMann) [inline packages/merge/src/components/MergeTable/MergeTable.js:64]: check API of Loadmask with required property an wrap the content

File: packages/merge/src/modules/merge/reducer.spec.js [hunk 3]
```diff
@@ -14,6 +14,114 @@ describe('merge', () => {
   describe('modules', () => {
     describe('modules', () => {
       describe('merge', () => {
+        describe('executeMerge', () => {
```
Comment (MethodenMann) [inline packages/merge/src/modules/merge/reducer.spec.js:17]: reducer is missing in describe path

File: packages/merge/src/components/Merge/Merge.js [hunk 4]
```diff
@@ -7,7 +7,7 @@ import MergeSummary from '../MergeSummary'
 
 const StyledMerge = styled.div`
   margin: 5px;
-  height: 100%;
+  height: 1000px;
```
[the remark this answers is not in the document]
Comment (stewue) [reply packages/merge/src/components/Merge/Merge.js:10]: I have just found out that this is no longer needed, as the action is now a fullscreen :D

**Review by MethodenMann — APPROVED** [review]

**Pull request merged**